Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1

```diff
@@ -102,22 +112,34 @@ function CreateRelativityVersionAsync() {
     $requestJson = $request | ConvertTo-Json
     Write-Message "Request Json: $($requestJson)"
 
-    $response = invoke-restmethod -uri $createrelativityversionasyncurl -method post -body $requestjson -contenttype 'application/json' -headers @{"x-csrf-header"="-"}
-    $responsejson = $response | convertto-json
-    Write-Message "Created Relativity Version"
+    if ($environmentName -eq 'Production') {
```

Comment:
instead of having 3 if conditions with the same code, we should probably loop all the 3 environments in a list or array. That way we have less duplicate code.

Reply:
Done